src/rastervision/common/models/densenet169.py

```diff
@@ -85,10 +90,10 @@ def DenseNet169(nb_dense_block=4, growth_rate=32, nb_filter=64, reduction=0.5,
 
     # load weights
     if weights == 'imagenet':
-        weights_path = '/opt/data/weights/densenet169_weights_tf.h5'
-        if not exists(weights_path):
+        path = join(weights_path, 'densenet169_weights_tf.h5')
+        if not exists(path):
             download_weights("densenet169_weights_tf.h5")
```

Comment:
Some more double quotes. Maybe you should do a search and replace on these files.